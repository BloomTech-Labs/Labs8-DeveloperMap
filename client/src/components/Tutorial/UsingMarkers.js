import React from 'react';
import company from '../../images/markerlogo4.png';
import seeker from '../../images/markerlogo.png';

import { MarkerGuide, Next } from './TutorialStyle';

class UsingMarkers extends React.Component {
  switchPage = () => {
    this.props.history.push('/tutorial/devprofileguide');
  };
  render() {
    return (
      <MarkerGuide>
        <h1>Using the Markers</h1>
        <p>
          There are two types of markers on the map. Companies are
          <span>
            <img src={company} alt="" style={{ width: '40px' }} />
          </span>
          and Job Seekers are
          <span>
            <img src={seeker} alt="" style={{ width: '40px' }} />
          </span>
          . You can click on a marker to bring up the name of the company or
          seeker in which that maker belongs to. Clicking on the link there will
          bring you to their profile page.
        </p>
        <p>
          If you want to display only seekers or only companies, you can use the
          key in the bottom right hand corner of your screen.
        </p>
        <h1>Picture Goes here</h1>
        <Next onClick={this.switchPage}>Next</Next>
      </MarkerGuide>
    );
  }
}

export default UsingMarkers;
