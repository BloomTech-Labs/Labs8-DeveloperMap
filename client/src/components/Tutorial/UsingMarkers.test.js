import React from 'react';
import renderer from 'react-test-renderer';
import UsingMarkers from './UsingMarkers';


describe(`using markers Component`, () => {
    it('renders correctly', () => {
        const tree = renderer.create(
          <UsingMarkers />
        ).toJSON();
        expect(tree).toMatchSnapshot();
      });

});

