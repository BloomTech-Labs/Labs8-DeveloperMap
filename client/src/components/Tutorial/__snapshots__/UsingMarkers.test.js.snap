// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`using markers Component renders correctly 1`] = `
<div
  className="sc-bwzfXH sc-EHOje fiNfSU"
>
  <h1>
    Using the Markers
  </h1>
  <p>
    There are two types of markers on the map. Companies are
    <span>
      <img
        alt=""
        src="markerlogo4.png"
        style={
          Object {
            "width": "40px",
          }
        }
      />
    </span>
    and Job Seekers are
    <span>
      <img
        alt=""
        src="markerlogo.png"
        style={
          Object {
            "width": "40px",
          }
        }
      />
    </span>
    . You can click on a marker to bring up the name of the company or seeker in which that maker belongs to. Clicking on the link there will bring you to their profile page.
  </p>
  <p>
    If you want to display only seekers or only companies, you can use the key in the bottom right hand corner of your screen.
  </p>
  <h1>
    Picture Goes here
  </h1>
  <button
    className="sc-bdVaJa fEBwJj"
    onClick={[Function]}
  >
    Next
  </button>
</div>
`;
